### PR TITLE
Add initial Google AdSense code for testing

### PR DESCRIPTION
**Pull request by rjungemann**

### CODE REVIEW

**Comment by Smittyvb** [comment]
This doesn't actually put any ads anywhere. Where do we plan to place ads?

**Comment by jredbeard** [comment]
> This doesn't actually put any ads anywhere. Where do we plan to place ads?

This is just an exploratory excercise for testing. The 'where' or even the 'if' is not decided.

File: src/server/server.js [hunk 1]
```diff
@@ -274,7 +282,39 @@ if (env === 'production') {
...
+            yield next;
+        } else {
+            // If user is signed out, enable ads.
+            let policy = this.response.header['content-security-policy']
+                .split(/;\s+/)
+                .map(el => {
+                    if (el.startsWith('script-src')) {
+                        const oldScriptSrc = el.replace(/^script-src/, '');
+                        `script-src 'unsafe-inline' 'unsafe-eval' data: http: https: ${
+                            scriptSrc
```
Comment (jredbeard) [inline src/server/server.js]: should this be `${oldScriptSrc}`?

Comment (rjungemann) [reply src/server/server.js]: Fixed

**Pull request closed without merging**